src/main/java/github/grace000/erversay/RouteHandlers/KittyImage.java

```diff
@@ -0,0 +1,57 @@
+package github.grace000.erversay.RouteHandlers;
+
+import github.grace000.erversay.Request.Request;
+import github.grace000.erversay.Response.Response;
+import github.grace000.erversay.Response.ResponseBuilder;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static github.grace000.erversay.Constants.Headers.*;
+import static github.grace000.erversay.Constants.StatusCodes.NOT_ALLOWED_STATUS;
+import static github.grace000.erversay.Constants.StatusCodes.OK_STATUS;
+
+public class KittyImage implements RouteHandler {
+    private ResponseBuilder responseBuilder = new ResponseBuilder();
+
+    public enum AcceptedMethods {
+        GET
+    }
+
+    public boolean isMethodAllowed(String method) {
+        for (AcceptedMethods acceptedMethods : AcceptedMethods.values()) {
+            if (acceptedMethods.name().equals(method)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public Response handle(Request request) {
+        ResponseBuilder fileResponse = responseBuilder;
+        if (isMethodAllowed(request.method)) {
+        try {
+            byte[] data = getFilePath();
+            fileResponse
+                    .withStatus(OK_STATUS)
+                    .withBody(data)
+                    .withHeaders(JPEG_IMAGE_HEADER);
```

Comment:
Any request or response with a non-empty body must have a `Content-Length` header along with a `Content-Type`. I recommend using a `Map` for response headers the same you do for request headers.